server/typedb.rs

```diff
@@ -227,6 +230,14 @@ fn synchronize_database_metrics(diagnostics_manager: Arc<DiagnosticsManager>, da
     diagnostics_manager.submit_database_metrics(metrics);
 }
 
+async fn resolve_address(address: String) -> SocketAddr {
+    lookup_host(address.clone())
+        .await
+        .unwrap()
+        .next()
+        .expect(format!("Unable to map address '{}' to any IP addresses", address).as_str())
```

Comment:
It should probably return an Error (`Result<SocketAddr>`) instead, but we can continue with this, but at least a todo.

Reply:
Given that this is not a generic method, but a specific one located in the server, I believe the method has enough context to decide to resolve it there and then.